### DISCUSSION TITLE
Dexcom one+ non compatible phone -> xdrip

### DISCUSSION THREAD

**Discussion by JensDh05** [NightscoutFoundation/xDrip#3552] [General]
Dear,

In a few days i'm mandatory to switch to dexcom one+ (instead of dexcom one).
I have a none compatible phone and use the BYO dexcom (one).
The problem is, the BYO dexcom is not available at the moment for the one+.

So i'm in search of a solution that i still can use my glucose monitorring (one+), without needing to buy a new phone.
Is this possible with xdrip?
If so, how?

thanks in advance.

Best regards
Jens

**Comment by Navid200** [comment]
Have you considered using xDrip to collect from One+ instead of using BYODA?
https://navid200.github.io/xDrip/docs/Dexcom/G7.html

**Comment by PhilTU** [comment]
I used a One+ sensor recently only with xDrip+ and nothing else. Worked well, followed Navid's instructions for G7 or One+ at link above

**Comment by JensDh05** [comment]
Thanks !
